cmd/system-probe/utils/utils.go

```diff
@@ -0,0 +1,19 @@
+package utils
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// WriteAsJSON marshals the give data argument into JSON and writes it to the `http.ResponseWriter`
+func WriteAsJSON(w http.ResponseWriter, data interface{}) {
```

Comment:
Not for this PR, but how can we merge this into our existing system-probe payload serialization logic that can handles JSON/PB?